packages/contracts-bedrock/contracts/universal/StandardBridge.sol

```diff
@@ -366,6 +372,9 @@ abstract contract StandardBridge {
             "StandardBridge: bridging ETH must include sufficient ETH value"
         );
 
+        // If the _emit function is overri theden it will emit legacy events before emitting the
+        // current Bridge event.
```

Comment:
I wanted to avoid using `super` because then you need to jump from the child contract to the parent to see what it does.

But I'll admit that this also looks weird. I can do that.